### ISSUE TITLE
StateToOCIState returns invalid pause status

### ISSUE THREAD

**Issue by zhsj** [kata-containers/kata-containers#1442]
https://github.com/kata-containers/kata-containers/blob/29d4abf23abf06b3419a862d6395ba86240975dc/src/runtime/virtcontainers/pkg/oci/utils.go#L1094

However OCI spec doesn't have paused status.

https://github.com/opencontainers/runtime-spec/blob/master/specs-go/state.go

It only has creating, created,  running, stopped.

**Comment by fidencio** [comment]
Adding a little bit of context: https://github.com/kata-containers/runtime/pull/328
Also, /cc @devimc

**Comment by zhsj** [comment]
I was looking the potential of bumping runtime-spec to latest one. The container status was changed from `string` to `ContainerState`.

I have checked the runc code, it has paused status, however it was never in the spec.

https://github.com/opencontainers/runc/blob/f245a1d1edbf545549e5a16106cf1aec356a3c7d/libcontainer/container.go#L26

https://github.com/opencontainers/runc/blob/f245a1d1edbf545549e5a16106cf1aec356a3c7d/libcontainer/container_linux.go#L1919 It's just converted to `ContainerState`, regardless of how the spec defines.
